### ISSUE TITLE
Program encodes some unnecessary files even with skipUnknown

### ISSUE THREAD

**Issue by TadeLn** [mdsitton/SngFileFormat#5]
If you have a file named `album_foobar.png` in the song folder, it will still be included in the .sng file, even with the `--skipUnknown` option. This is also the case for highways, backgrounds, and videos.

I'm not really sure if this is a bug or intended behavior, but it seems like a bug.

The bug seems to be [here](https://github.com/mdsitton/SngFileFormat/blob/18ae0169b150036d311dbd0e2b9056afdfc00b77/SngTool/SngCli/SngEncode.cs#L556). The filename is checked with StartsWith instead of checking the whole filename up to the last `.`

**Comment by mdsitton** [comment]
I'd say this is a bug yes, thanks for the report. i'll get this fixed soon

**Comment by TadeLn** [comment]
I just found out that CH allows for multiple backgrounds apparently, so I think this behavior is fine for background images, but it shouldn't be the case for album art. I don't know what about highways and background videos, I can't check that right now... I'll check it in a few hours when I get home.

**Comment by TadeLn** [comment]
Okay, checked with CH v1.1.0 PTB:
- Album art: only a single file named `album` (with the correct file extension) works
- Background videos: only a single file named `video` (with the correct file extension) works
- Highways: only a single file named `highway` (with the correct file extension) works
- Background images: multiple files are allowed, the game chooses randomly from one of the files named `background*` (with the correct file extension) where `*` is anything (including nothing)

Which means that this bug should be fixed for album arts, videos, and highways, but backgrounds work fine currently.

**Comment by mdsitton** [comment]
> I just found out that CH allows for multiple backgrounds apparently, so I think this behavior is fine for background images, but it shouldn't be the case for album art. I don't know what about highways and background videos, I can't check that right now... I'll check it in a few hours when I get home.

Yes that is true but for ch such backgrounds have to contain background (casing doesn't matter) in the filename for it to be considered as a background.
